test/services/exposed/error-handler.test.js

```diff
@@ -108,8 +120,37 @@ describe('services › exposed › error-handler', () => {
       errors: [{
         detail: 'Something bad happened',
         status: 666,
-        name: 'SomethingWrongHappenedError',
+        name: 'Error',
       }],
     }]);
   });
+
+  describe('with an error that supports data (payload)', () => {
```

Comment:
Can you add a tests with a VError ? I don't think a case with an error with an array of errors in it.

Reply:
Do you talk about retrieving data from VError? I've removed the code, It's not supported anymore as we only have 2 errors using VError. IMO that thing should be removed ASAP.

Reply:
My point was that the coded that is added by this PR works for errors having a property "error" with an array of errors, and I don't see a test for this case in unit tests. I assumed that it was the case of `VError`, but I'm not sure.

Is it possible to add tests for all the cases that are supported by the code, including the property `error`?

Reply:
I forgot about that one.

`VError` cannot transport data property if that is what you meant. But I can add the support for this even if I would prefer simply removing `VError`.